.travis.yml

```diff
@@ -8,6 +8,7 @@ install:
   - make install-travis
 
 script:
+  - OS_GOFLAGS="-p 1" travis_wait make build
```

Comment:
not sure this single line is really helpful. I want to avoid that `-p1` is passed to every command in verify. But, of course this has the overhead that we call go-build/install twice, although caching is involved.